scAR-cpu.yml

```diff
@@ -0,0 +1,23 @@
+name: scAR
+channels:
+- nvidia
+- pytorch
+- conda-forge
+dependencies:
+- conda-forge::python>=3.8.6
+- nvidia::cudatoolkit>=11.1
+- pytorch::pytorch>=1.10.0
+- pytorch::tcorchvision>=0.9.0
+- pytorch::torchaudio>=0.8.0
+- pytorch-mutex=*=cpu
```

Comment:
`pytorch-mutex` is a conda metapackage that you can use to instruct conda to install the cpu or the cuda (gpu) version of the package.